update bug report template (#10807)
netdata/netdata: .github/ISSUE_TEMPLATE/bug_report.md

##### Netdata version
<!--
Provide output of `netdata -W buildinfo`.

> In case your agent is older and doesn't support buildinfo do following:

Reviewer: ```suggestion
If your Netdata Agent is older, and doesn't support buildinfo, do the following:
```

Author: Thank you :)

Reviewer 2: > do the following

update Netdata and re-test 😄
